### PR TITLE
change delimiter

### PR DESCRIPTION
**Pull request by emya**
[#150654816]

### CODE REVIEW

File: lib/etl/redshift/client.rb [hunk 1]
```diff
@@ -151,9 +150,9 @@ def delete_object_from_s3(bucket, prefix, session_name)
 
     # Upserts rows into the destintation tables based on rows
     # provided by the reader.
-    def upsert_rows(reader, table_schemas_lookup, transformer, delimiter='|')
-      # write csv files
-      arr = write_csv_files(reader, table_schemas_lookup, transformer, delimiter)
+    def upsert_rows(reader, table_schemas_lookup, transformer, delimiter='\\001')
+      # write csv file
```
Comment (ChrisPRobinson) [inline lib/etl/redshift/client.rb]: Nit: its actually writing files not file, I would keep the comment the same

File: lib/etl/redshift/client.rb [hunk 2]
```diff
@@ -151,9 +150,9 @@ def delete_object_from_s3(bucket, prefix, session_name)
...
+      arr = write_csv_files(reader, table_schemas_lookup, transformer)
```
Comment (ChrisPRobinson) [inline lib/etl/redshift/client.rb]: The delimiter is passed in but is then not passed down, so if its different that \\001 then it won't be passed correctly

Comment (emya) [reply lib/etl/redshift/client.rb]: The delimiter is used for `copy_from_s3`. We need to use `\u0001` when write into files, and `\\001` for loading into Redshift. 
Actually I don't think we have to have a parameter for delimiter anymore.
We should remove all delimiter parameters.

Comment (ChrisPRobinson) [reply lib/etl/redshift/client.rb]: ok, lets remove it then, it wasn't clear from the code that was what was going on

Comment (emya) [reply lib/etl/redshift/client.rb]: Sorry I misunderstood. `\u0001` works on both cases. I thought ASCII characters have to be in octal in Copy command, but we can use ASCII as is.  Anyway, we don't have to switch the delimiter, so I gonna remove them.

**Pull request merged**